src/lib/core/a11y/interactivity-checker.ts

```diff
@@ -15,8 +16,13 @@ export class InteractivityChecker {
 
   constructor(private _platform: Platform) {}
 
-  /** Gets whether an element is disabled. */
-  isDisabled(element: HTMLElement) {
+  /**
+   * Gets whether an element is disabled.
+   *
+   * @param element Element to be checked.
+   * @returns {boolean} Whether the element is disabled.
```

Comment:
We do, dgeni doesn't figure out the return types, it was requested by @jelbourn.